### PR TITLE
[Refactor-261] review and remove comments from prs

### PR DESCRIPTION
**Pull request by daniel-ilan**
Deleted all redundant comments

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -31,10 +31,7 @@ const App: FC = () => {
 
   const appDir = i18n.dir();
   useEffect(() => {
-    // https://material-ui.com/guides/right-to-left/
```
Comment (yuvalbl) [inline src/App.tsx:34]: these comments are OK, and should not be removed

File: src/components/molecules/OverlayLoader.tsx [hunk 2]
```diff
@@ -25,7 +25,6 @@ interface IProps {
 }
 
 const OverlayLoader: FC<IProps> = ({ show }) => {
-  // passing props to useStyles - see demo: https://codesandbox.io/s/giubj?file=/demo.js:224-229
```
Comment (yuvalbl) [inline src/components/molecules/OverlayLoader.tsx:28]: this comment is OK, and should not be removed

File: src/components/molecules/Table/TableView.tsx [hunk 3]
```diff
@@ -37,9 +37,7 @@ const StyledTableCell = withStyles((theme: Theme) =>
 )(TableCell);
 const StyledTableRow = withStyles(() =>
   createStyles({
-    root: {
-      // styles can be added here
```
Comment (yuvalbl) [inline src/components/molecules/Table/TableView.tsx:41]: this comment is OK, and should not be removed

File: src/components/molecules/Table/formatTableData.service.ts [hunk 4]
```diff
@@ -12,22 +11,16 @@ export const createTableData = (
 ) => {
   const arrayOfItems = items
     .map((item) => {
-      return (
-        Object.keys(item)
-          //filter by data keys
```
Comment (yuvalbl) [inline src/components/molecules/Table/formatTableData.service.ts:17]: these comments are OK, and should not be removed

File: src/components/molecules/Table/formatTableData.service.ts [hunk 5]
```diff
@@ -12,22 +11,16 @@ export const createTableData = (
...
+        .filter((key) => dataKeys.includes(key))
+        .sort((a, b) => dataKeys.indexOf(a) - dataKeys.indexOf(b))
+        .reduce((obj: object, key: string) => {
+          return {
+            ...obj,
+            [key]: item[key],
+          };
+        }, {});
     })
-    //create array of data arrays
```
Comment (yuvalbl) [inline src/components/molecules/Table/formatTableData.service.ts:30]: this comment is OK, and should not be removed

File: src/components/molecules/card/AnyWayCard.tsx [hunk 6]
```diff
@@ -37,9 +37,9 @@ const getSizeFactor = (options: CardSizeOptions | undefined): number => (options
 const useStyles = makeStyles((theme) => ({
   root: {
     fontFamily: fontFamilyString,
-    position: 'relative', // for meta tags
```
Comment (yuvalbl) [inline src/components/molecules/card/AnyWayCard.tsx:40]: these comment is OK, and should not be removed

File: src/services/to-image.service.ts [hunk 7]
```diff
@@ -19,8 +19,6 @@ const widgetToImage = (fileName: string, widgetElement: HTMLElement, scale = DEF
     usingDomToImage(fileName, widgetElement, scale);
   }
 };
-// Uses canvas. OK for maps, buggy for other elements
```
Comment (yuvalbl) [inline src/services/to-image.service.ts:22]: this comment is OK, and should not be removed

**Review by eidan66 — APPROVED** [review]

**Review by yuvalbl — APPROVED** [review]

**Pull request merged**